addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -83,6 +83,7 @@ function factory(dependencies) {
         }
 
         showSendMessage() {
+            this.thread.fetchUpdateSuggestedPartners();
```

Comment:
Async fetch of suggested recipient and immediate display of composer would imply some feedback that suggested recipients are being loaded. Either consider this in this task, or make a task for future improvement.